frontend/src/component/feature/FeatureStrategy/FeatureStrategyMenu/FeatureStrategyMenuCard/FeatureStrategyMenuCard.tsx

```diff
@@ -90,14 +104,18 @@ export const FeatureStrategyMenuCard = ({
             <StyledIcon>
                 <StrategyIcon />
             </StyledIcon>
-            <div>
+            <StyledContentContainer>
                 <StyledName
                     text={strategy.displayName || strategyName}
                     maxWidth='200'
                     maxLength={25}
                 />
-                <StyledDescription>{strategy.description}</StyledDescription>
-            </div>
+                <Tooltip title={strategy.description} arrow placement='top'>
```

Comment:
Making the content of description to now expand across multiple lines and adding tooltip.